components/pd_client/src/util.rs

```diff
@@ -425,8 +427,10 @@ pub async fn try_connect_leader(
     let members = previous.get_members();
     let cluster_id = previous.get_header().get_cluster_id();
     let mut resp = None;
+    let mut shuffle_members: Vec<_> = members.to_vec();
```

Comment:
```suggestion
    let mut shuffle_members = members.to_vec();
```

I think the type annotation here is unnecessary.